styles/globals.css

```diff
@@ -259,3 +259,61 @@ backdrop-filter: blur(10px); */
 .hoverEffect:hover {
   transform: scale(1.1); /* Scale the image by 10% on hover */
 }
+
+/* Back to Top Button Styles */
+.back-to-top-button {
+  position: fixed;
+  bottom: 30px;
+  right: 30px;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  background-color: rgb(20, 20, 20);
+  border: none;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 0px 0px 0px 4px rgba(180, 160, 255, 0.253);
+  cursor: pointer;
+  transition: all 0.3s ease;
+  overflow: hidden;
+  z-index: 1000;
+  outline: none;
+}
+
+.back-to-top-icon {
+  width: 12px;
+  height: 12px;
+  transition: all 0.3s ease;
+  fill: white;
+}
+
+.back-to-top-text {
+  position: absolute;
+  color: white;
+  font-size: 0px;
+  opacity: 0;
+  transition: all 0.3s ease;
+  white-space: nowrap;
+}
+
+.back-to-top-button:hover {
+  width: 140px;
+  border-radius: 50px;
+  background-color: rgb(181, 160, 255);
+}
+
+.back-to-top-button:hover .back-to-top-icon {
+  transform: translateY(-200%);
+  opacity: 0;
+}
+
+.back-to-top-button:hover .back-to-top-text {
+  font-size: 13px;
+  opacity: 1;
+}
+
+.back-to-top-button:active {
+  transform: scale(0.95);
+}
```

Comment:
Since we’re already using Tailwind in the project, I think we can handle some of the button styling directly with Tailwind class names.